driver/src/main/java/org/neo4j/driver/internal/InternalEntity.java

```diff
@@ -32,20 +32,25 @@
 
 public abstract class InternalEntity implements Entity, AsValue
 {
+    public static final String INVALID_ID_ERROR = "Numeric id is not available, please use string based element id alternative";
```

Comment:
Did we settle on the content of the error message? I'm worried that refering to it as "Numeric" might be confusing. How about: "id() is not available for this storage engine, please use elementId() instead."

Reply:
Updated the message. It needs to remain flexible enough to be used in multiple methods.